src/charm.py

```diff
@@ -229,18 +251,17 @@ def _set_version(self) -> None:
     def _admin_ui_pebble_layer(self) -> Layer:
         """Define pebble layer."""
         container_env = {
-            "KRATOS_ADMIN_URL": self._get_kratos_endpoint_info("admin_endpoint"),
-            "KRATOS_PUBLIC_URL": self._get_kratos_endpoint_info("public_endpoint"),
+            "KRATOS_ADMIN_URL": self._get_kratos_info("admin_endpoint"),
```

Comment:
Have a question about repeatedly calling the methods `_get_xxx_info` in this method.

Is it better that we just call `_get_xxx_info` once and get each value by keys?

Reply:
Good points, thanks